src/server/rpc/procedures/geolocation/geolocation.js

```diff
@@ -170,14 +176,16 @@ if(!process.env.GOOGLE_GEOCODING_API) {
             requestOptions.qs.keyword = keyword;
         }
 
-        return rp(requestOptions).then(res=>{
-            let places = res.results;
-            places = places.map(place => {
-                return [['latitude',place.geometry.location.lat],['longitude',place.geometry.location.lng],['name',place.name],['types',place.types]];
+        return cache.wrap(coordsToCacheKey() + keyword + radius, () => {
```

Comment:
why aren't you passing the lat/long? Is this intentional?

Reply:
nope, I'm adding some test..